docs/usage/self-hosted-configuration.md

```diff
@@ -324,28 +324,96 @@ This private key is used to decrypt config files.
 The corresponding public key can be used to create encrypted values for config files.
```

Comment:
The rich diff preview is failing, I'm not sure what the reason for that is...

Reply:
Works for me

Reply:
It's working for me now as well. 😄 Weird...